### PR TITLE
maintainers: Add wifi tag for hostap files

### PR DESCRIPTION
**Pull request by jukkar**
Make sure we run wifi tests for all hostap related files that are changed.

### CODE REVIEW

**Review by pdgendt — DISMISSED** [review]

File: MAINTAINERS.yml [hunk 1]
```diff
@@ -5156,6 +5156,8 @@ West:
     - modules/hostap/
   labels:
     - "area: Wi-Fi"
+  tests:
+    - wifi
```
Comment (rlubos) [inline MAINTAINERS.yml:5160]: Shouldn't this be `net.wifi` as it is for the `"Networking: Wi-Fi":` area?

Comment (jukkar) [reply MAINTAINERS.yml:5160]: But in the wifi tests, we have `wifi` tag, shouldn't it match here. Unfortunately there is no documentation at the start of the file about how this `tests` section should be filled.

Comment (rlubos) [reply MAINTAINERS.yml:5160]: Actually, I'm not sure anymore. Most of the entries seem to refer to test identifiers, but `trusted-firmware-m` for example does seem to refer to tag.

**Comment by jukkar** [comment]
To reviewers, please take a look if what I am proposing is correct. If this works as intended (wifi tests are run if PR touches hostap files), then I would like this to be merged before any pending wifi PRs in order to validate them properly.

**Review by nashif — CHANGES_REQUESTED** [review]
this should not be listing tags, this should be the test identifier, so `net.wifi` is the correct entry. Also note that those are still not being used anywhere for now, there is a PR that will activate them.
Probably you want to change this: scripts/ci/tags.yaml

and add any wifi related paths to the net entry, so networking tests do not get skipped or add a new entry for wifi

**Review by nashif — APPROVED** [review]

**Review by rlubos — APPROVED** [review]

**Pull request merged**